runtime/doc/helphelp.txt

```diff
@@ -438,6 +438,13 @@ also implicitly stops the block of ex-commands before it.  E.g. >
     endfunction
 <
 
+To add annotation in the block, place the annotation (ex: vim) after a greater
```

Comment:
This makes it sound like you can specify an arbitrary language (as in Nvim), but from the syntax file changes it looks like like _only_ `vim` is supported (which is probably enough for Vim)?

Reply:
I think it may make sense to also support python for `:h if_pyth.txt`, lua for `if_lua.txt`, perl for `if_perl.txt`, ruby for `if_ruby.txt`, `scheme` (?)  for `if_mzsch.txt` and `tcl` for `if_tcl.txt` ?

Reply:
And bash for shell snippets, if you want to go all the way.

Reply:
not sure if we have bash snippets in the helpfile

Reply:
I was thinking of code snippets that show launching Vim from the shell, not necessarily full bash scripts. But I don't know if there in fact are some. (There are in Nvim, but not many.)

Reply:
Should I add other language supports?

It is possible to implement.  But it includes other language syntax files.
It affects the performance.
I don't think it is used many...

Reply:
I can add lua support.  But I think lua is not used well in Vim plugins.
It is useful for Neovim though.

Reply:
(It is not very useful in Neovim, since we alwaysuse treesitter for help files, which gives us all these syntax injections for free.)